_authentication.md

```diff
@@ -267,5 +267,5 @@ curl https://api.uphold.com/v0/me \
 
 You can use Basic Authentication by providing your email and password combination.
 
-If OTP (One-Time Password, also known as Two-Factor Authentication) is required, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `OTP-Token: Required` and/or `OTP-Method-Id: Required`.
+If OTP (One-Time Password, also known as Two-Factor Authentication) is required, then you will get a [401 HTTP error](#errors), along with the HTTP header `OTP-Token: Required` and/or `OTP-Method-Id: Required`.
```

Comment:
@waldyrious would be good to clarify which one do we show and under which scenario.

Reply:
Actually, both need to be passed together, so we should just replace `and/or` with `and` here. I'll do so in a bit.

Reply:
[Done](https://github.com/uphold/docs/compare/9e692daf5f8cb6545f978a21f478f947d13867cd..abf9dc5ec31154234ea42eb42040d3520c3af053#diff-abc260fb4e7cf8b00d11e36bbf570d59).